### ISSUE TITLE
A planning group filled with joints doesn't work, same group defined with chain works fine.

### ISSUE THREAD

**Issue by hersh** [moveit/moveit_ros#335]
Following the tutorial http://moveit.ros.org/wiki/PR2/Setup_Assistant/Quick_Start I created a moveit config for PR2 which did not work.  When I compared the resulting config with the official pr2_moveit_config, I saw that the tutorial had me select a list of joints for each arm group, but the official one had specified a chain for each arm group.  When I went back and re-did one arm of my moveit config using the chain, that one arm ended up working.

Bug: specifying a list of joints should work.

**Comment by isucan** [comment]
Was this with latest source code in ROS Hydro?

**Comment by hersh** [comment]
yes, it was with the latest.  hydro-devel branch.  I followed the instructions for a source code checkout on the wiki (source of moveit) and this on top of a hydro ROS installation. (but public ros, not shadow-fixed).

**Comment by isucan** [comment]
This is likely a bug in the chain detection method. This would be in the robot_model library, in moveit_core.
